### DISCUSSION TITLE
java.awt.GraphicsDevice and getIDString method

### DISCUSSION THREAD

**Discussion by gv-csgroup** [adoptium/adoptium-support#481] [Q&A] [answered]
Hello,

I need to open several Frame from different JVM processes on a specific screen. I.E. a screen where a main JFrame is already opened.

My intention was to use this method to identify the screen https://devdocs.io/openjdk~11/java.desktop/java/awt/graphicsdevice#getIDstring().

But I am wondering whether it is a reliable method as the javadoc indicates it is intended for "debugging pruposes".

May I rely on its value to identify the screen a JFrame is opened on, and pass it as an argument to a second JVM ?

My environment is AdoptOpendJDK11 on windows

An suggestion would be helpful ,

thanks

**Comment by karianna** [comment, marked as answer]
Apologies but we don't offer general programming advice via this forum.  you can try our slack community adoptium.net/slack.
